docs/guides/admin/docs/installation/example-installation.md

```diff
@@ -0,0 +1,361 @@
+
+    org.opencastproject.server.url=http://<URL>:8080
+
+Set a secure password for the admin account and the digest user:
+
+    org.opencastproject.security.admin.pass=CHANGE_ME
+
+    org.opencastproject.security.digest.pass=CHANGE_ME
+
+
+Set the location of the shared storage directory:
+
+    org.opencastproject.storage.dir=/srv/opencast
+
+Set the database URL and user credentials. Use the DB name and password you set up earlier when creating the DB:
+
+    org.opencastproject.db.jdbc.url=jdbc:mariadb://admin.example.com/opencast?useMysqlMetadata=true
+    org.opencastproject.db.jdbc.user=opencast
+    org.opencastproject.db.jdbc.pass=dbpassword
+
+Next up open the file `/etc/opencast/org.opencastproject.organization-mh_default_org.cfg`. Here you need to configure the URLs to the respective Opencast nodes.
+
+Set the link to the AdminUI which is part of the admin node:
+
+    prop.org.opencastproject.admin.ui.url=http://admin.example.com:8080
+
+Set the link to the engage ui which essentially means the presentation node:
+
+    prop.org.opencastproject.engage.ui.url=http://presentation.example.com:8080
+
+This should be enough to be able to successfully start Opencast and start testing its features.
+
+Start Opencast
+--------------
+
+Finally, start and enable Opencast by running:
+
+```sh
+systemctl start opencast.service
+systemctl enable opencast.service
+```
+
+Before trying to login you should check, if the startup was successful. You can do so by running:
+
+```sh
+systemctl status opencast.service
+```
+
+If the status says "running" and no errors are to be seen, you should now be able to go to https://admin.example.com , login with your specified credentials and start uploading videos. If everything works as expected you can also start checking out the plethora of possibilities and configuration options that Opencast provides. In general it is advisable to only change one functonality at a time in order to test if the configuration was successfull. If the admin-ui does not show up or anything unexpected happens, refer to the next section.
+
+Troubleshooting
+------------
+If there are any problems with starting up Opencast, accessing a video or any errors during processing of a video a good place to start your investigation are the most recent Opencast logs which are located by default in `/var/logs/opencast/opencast.log`. Open the file, find the first error message that appeared and check what it says.
+In most cases, if there are some faulty configurations, files missing or connection problems there will be a corresponding message which should lead you to the solution.
+
+If Opencast is running, but the AdminUI does not show up you should also check if nginx is started and the configuration file is valid. Invalid server certificates may also be an issue.
+
+If you can not solve the problem on your own you can also always ask in the appropriate Opencast mailing lists or community chatrooms.
```

Comment:
Link to the relevant docs here.  That might be the general docs.opencast.org landing page, or it might be something in here, not sure.